src/cmd/cmd_sym.ml

```diff
@@ -58,14 +58,18 @@ let cmd profiling debug unsafe optimize workers no_stop_at_failure no_values
   let* _created_dir = Bos.OS.Dir.create ~path:true ~mode:0o755 workspace in
   let pc = Choice.return (Ok ()) in
   let result = List.fold_left (run_file ~unsafe ~optimize) pc files in
-  let thread : Thread.t = Thread.create () in
+  let thread = Thread_with_memory.init () in
   let res_queue = Wq.init () in
-  let callback = function
-    | Symbolic_choice.EVal (Ok ()), _ -> ()
+  (* Hackity hack *)
+  let path_counter = ref 0 in
```

Comment:
It seems like this is incremented in `callback` but never read ?

Reply:
Ah oops, I did this just to debug a thing. I'll remove it